packages/env/lib/config/parse-config.js

```diff
@@ -117,11 +117,21 @@ function parseSourceString( sourceString, { workDirectoryPath } ) {
 	const gitHubFields = sourceString.match(
 		/^([^\/]+)\/([^#\/]+)(\/([^#]+))?(?:#(.+))?$/
 	);
+
+	if (
+		gitHubFields &&
+		! gitHubFields[ 5 ] &&
+		gitHubFields[ 1 ] === 'WordPress'
+	) {
+		gitHubFields[ 5 ] =
+			gitHubFields[ 2 ] === 'WordPress' ? 'master' : 'trunk';
```

Comment:
Essentially, every WordPress org repo _except_ the `WordPress` repository gets `trunk` as default? Seems fair enough, though I imagine there are more edge cases. It probably doesn't make sense to cover all of them :) 

Unless there's a way to use the GitHub API to easily figure out a remote repo's primary branch?

Actually, having said that, doesn't `git clone https://github.com/WordPress/gutenberg.git` automatically use the correct base branch? I wonder if it'd then be possible to avoid setting a ref whatsoever if none is given. Then we don't have to worry about base branches at all. 

The only concern is that I don't know if the git wrapper handles that.

Reply:
> Actually, having said that, doesn't git clone https://github.com/WordPress/gutenberg.git automatically use the correct base branch

Ah, that would be a better approach, will give it a try tomorrow and see if the wrapper does handle it or not.

Reply:
@noahtallen, it seems to work to not specify a fallback and just let git fall back to the repos default branch 🎉